modules/ensemble/lib/widget/button.dart

```diff
@@ -149,6 +155,24 @@ class ButtonState extends EWidgetState<Button> {
   }
 
   Widget _buildButtonChild() {
+    if (widget._controller.loading == true) {
+      return widget._controller.loadingWidget != null && scopeManager != null
+          ? scopeManager!
```

Comment:
@vusters does this look ok to you, I know you changed how widgets are created recently?